fmc/src/flow/rt_alias.rs

```diff
@@ -104,6 +106,13 @@ impl RtAliasLayer {
         Self::extend_pcrs(env, hand_off)?;
         cprintln!("[alias rt] Extend RT PCRs Done");
 
+        cprintln!("[alias rt] Lock RT PCRs");
+        env.pcr_bank
+            .set_pcr_lock(caliptra_common::RTALIAS_CURRENT_PCR);
```

Comment:
Nit: "alias" is a DICE concept, but these PCRs are a different thing. Suggest calling it "RT_FW_CURRENT_PCR" or something.

Reply:
done